src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Customizations/Indexers/IIndexersOperations.Customization.cs

```diff
@@ -26,11 +27,26 @@ public partial interface IIndexersOperations
         /// Additional parameters for the operation
         /// </param>
         /// <param name='customHeaders'>
-        /// The headers that will be added to request.
+        /// Headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
+        /// <exception cref="CloudException">
+        /// Thrown when the operation returned an invalid status code.
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.SerializationException">
+        /// Thrown when unable to deserialize the response.
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.ValidationException">
+        /// Thrown when a required parameter is null.
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null.
+        /// </exception>
+        /// <return>
```

Comment:
The tag is set to 'returns' at quite a few places. Please correct these.

Reply:
It looks like an issue with the generated code that copied over to the custom code. It's supposed to be `<returns>` everywhere. I'll fix it in the custom code. In the meantime, I've filed an issue on AutoRest: https://github.com/Azure/autorest/issues/2957